### PR TITLE
Feature/UI nft plugin

### PR DESCRIPTION
**Pull request by vinhnt-dn**

### CODE REVIEW

File: src/view/nftPlugin/cardNFT/imageNFT.tsx [hunk 1]
```diff
@@ -1,23 +1,14 @@
...
 
-import IMAGE_DEFAULT from 'static/images/nft.jpeg'
+import IMAGE_DEFAULT from 'static/images/nft-default.svg'
 
 type ImageNFTProps = {
   src?: string
   size?: number
 }
 
 const ImageNFT = ({ src, size = undefined }: ImageNFTProps) => {
```
Comment (ghost) [inline src/view/nftPlugin/cardNFT/imageNFT.tsx]: No need for `size = undefined`

Comment (ghost) [inline src/view/nftPlugin/cardNFT/imageNFT.tsx]: we should add the default of `src` in input instead

File: src/view/nftPlugin/cardNFT/imageNFT.tsx [hunk 2]
```diff
@@ -1,23 +1,14 @@
-import { Image } from 'antd'
+import { Avatar } from 'antd'
 
-import IMAGE_DEFAULT from 'static/images/nft.jpeg'
+import IMAGE_DEFAULT from 'static/images/nft-default.svg'
 
 type ImageNFTProps = {
```
Comment (ghost) [inline src/view/nftPlugin/cardNFT/imageNFT.tsx]: always export type as a team convention

File: src/view/nftPlugin/cardNFT/imageNFT.tsx (hunk 1 above)
Comment (vinhnt-dn) [reply src/view/nftPlugin/cardNFT/imageNFT.tsx]: I have fixed. Please check again. Thank you sir!

Comment (vinhnt-dn) [reply src/view/nftPlugin/cardNFT/imageNFT.tsx]: @tuphan-dn 
 I have fixed. Please check again. Thank you sir!

File: src/view/nftPlugin/cardNFT/imageNFT.tsx (hunk 2 above)
Comment (vinhnt-dn) [reply src/view/nftPlugin/cardNFT/imageNFT.tsx]: @tuphan-dn I have fixed. Please check again. Thank you sir!

**Pull request merged**